Added Release Notes v5.7 (#106)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/release-notes.adoc

- Add Near Cache support for Map CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/661[#661])
- Add the ability to set arbitrary JVM arguments (https://github.com/hazelcast/hazelcast-platform-operator/pull/653[#653])
- Enables to set JVM Garbage Collector (GC) configuration using Hazelcast Platform Operator (https://github.com/hazelcast/hazelcast-platform-operator/pull/647[#647])
- Support `-XX:{Initial,Max,Min} RAMPercentage` Java options (https://github.com/hazelcast/hazelcast-platform-operator/pull/567[#567])
- Native memory support for Cache CR becomes available for the user (https://github.com/hazelcast/hazelcast-platform-operator/pull/645[#645])
- Add support for creating route resources in OpenShift environments for Management Center (https://github.com/hazelcast/hazelcast-platform-operator/pull/631[#631])
- Add Ingress support for Management Center external connectivity (https://github.com/hazelcast/hazelcast-platform-operator/pull/608[#608])
- Add support for advanced networking configuration through the Hazelcast Platform Operator (https://github.com/hazelcast/hazelcast-platform-operator/pull/605[#605])

> - Enables Management Center configuration in the Hazelcast cluster monitoring (https://github.com/hazelcast/hazelcast-platform-operator/pull/675[#675])

Author: ```suggestion
- Enables the configuring of Management Center monitoring parameters (https://github.com/hazelcast/hazelcast-platform-operator/pull/675[#675])
```

> - Enables Management Center configuration in the Hazelcast cluster monitoring (https://github.com/hazelcast/hazelcast-platform-operator/pull/675[#675])

Reviewer: Yes that looks fine.

> - Enables the configuring of Management Center monitoring parameters (https://github.com/hazelcast/hazelcast-platform-operator/pull/675[#675])

Reviewer: ```suggestion
- Enabled the configuring of Management Center monitoring parameters (https://github.com/hazelcast/hazelcast-platform-operator/pull/675[#675])
```

== Changes
- Remove support for `hostPath` in Hazelcast persistence (https://github.com/hazelcast/hazelcast-platform-operator/pull/607[#607])
- Define the default port of endpoints when not declared by the user in WAN Replication (https://github.com/hazelcast/hazelcast-platform-operator/pull/662[#662])
- Trigger WAN Replication for newly created Map CR in Hazelcast resource (https://github.com/hazelcast/hazelcast-platform-operator/pull/656[#656])
- Add an external address for Ingress and Route in Management Center (https://github.com/hazelcast/hazelcast-platform-operator/pull/643[#643])

> - Improved map deletion from WAN status when publisher ID is empty and improved logs (https://github.com/hazelcast/hazelcast-platform-operator/pull/615[#615])

Reviewer: What about this:

Maps without a `publisherID` are now deleted from the WAN status to resolve an issue where the finalizer did not exit successfully.

> - Improved map deletion from WAN status when publisher ID is empty and improved logs (https://github.com/hazelcast/hazelcast-platform-operator/pull/615[#615])

Author: ```suggestion
- Maps without a `publisherID` are now deleted from the WAN status to resolve an issue where the finalizer did not exit successfully (https://github.com/hazelcast/hazelcast-platform-operator/pull/615[#615])
```

> - Update CSV file for multi-namespace and cluster-wide operator support (https://github.com/hazelcast/hazelcast-platform-operator/pull/599[#599])

Author: ```suggestion
- Updated the cluster service version (CSV) file for multi-namespace and cluster-wide operator support (https://github.com/hazelcast/hazelcast-platform-operator/pull/599[#599])
```

- Add option to create namespace-scoped cluster role for Hazelcast Platform Operator (https://github.com/hazelcast/hazelcast-platform-operator/pull/594[#594])
- Add cluster-scoped operator support to Helm chart (https://github.com/hazelcast/hazelcast-platform-operator/pull/589[#589])
- Add option for operator installation with namespace scoped roles in Helm chart (https://github.com/hazelcast/hazelcast-platform-operator/pull/- 588[#588])

> - Move WebHook CA bundle addition logic to Helm (https://github.com/hazelcast/hazelcast-platform-operator/pull/586[#586])

Author: ```suggestion
- The Webhook installation becomes configurable in the Helm chart values field `webhook.enabled`(https://github.com/hazelcast/hazelcast-platform-operator/pull/586[#586])
```

> - Move WebHook CA bundle addition logic to Helm (https://github.com/hazelcast/hazelcast-platform-operator/pull/586[#586])

Author: It might be better. What do you think? @angelasimms

> - Move WebHook CA bundle addition logic to Helm (https://github.com/hazelcast/hazelcast-platform-operator/pull/586[#586])

Reviewer: The Webhook installation is now configurable in the Helm chart values field `webhook.enabled`

> - Added a new installation option for using the Hazelcast Platform Operator with namespace-scoped roles in Helm chart (https://github.com/hazelcast/hazelcast-platform-operator/pull/- 588[#588])

Reviewer: ```suggestion
- Added a new installation option for using the Hazelcast Platform Operator with namespace-scoped roles in Helm chart (https://github.com/hazelcast/hazelcast-platform-operator/pull/588[#588])
```

- Change `podManagementPolicy` to `Parallel` to improve cluster setup time (https://github.com/hazelcast/hazelcast-platform-operator/pull/539[#539])
- Support for the cluster-scoped operator is improved in the phone home package and Kubebuilder roles (https://github.com/hazelcast/hazelcast-platform-operator/pull/521[#521])
- Update Hazelcast Go client to 1.4.0 (https://github.com/hazelcast/hazelcast-platform-operator/pull/663[#663])
- Update default agent version to 0.1.16 (https://github.com/hazelcast/hazelcast-platform-operator/pull/690[#690])
- Add phone home for watched namespace type becomes available (https://github.com/hazelcast/hazelcast-platform-operator/pull/616[#616])

== Fixed Issues
- Default WAN Replication is available through Hazelcast service on port 5710, and if no serviceType is specified during configuration, it will be set to LoadBalancer (https://github.com/hazelcast/hazelcast-platform-operator/pull/706[#706])
- Make Hazelcast container ports dynamically updatable and remove infinite triggers (https://github.com/hazelcast/hazelcast-platform-operator/pull/692[#692])

> - Fix `CreateOrUpdate` function usage to prevent Map CR from being deleted accidentally (https://github.com/hazelcast/hazelcast-platform-operator/pull/686[#686])

Reviewer: ```suggestion
- Fixed `CreateOrUpdate` function usage to prevent the accidental deletion of the Map CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/686[#686])
```
